moveit_ros/planning/planning_scene_monitor/src/current_state_monitor.cpp

```diff
@@ -136,21 +138,29 @@ void planning_scene_monitor::CurrentStateMonitor::clearUpdateCallbacks()
 
 void planning_scene_monitor::CurrentStateMonitor::startStateMonitor(const std::string& joint_states_topic)
 {
+  rclcpp::Clock ros_clock;
   if (!state_monitor_started_ && robot_model_)
   {
     joint_time_.clear();
     if (joint_states_topic.empty())
-      ROS_ERROR("The joint states topic cannot be an empty string");
+    {
+      RCLCPP_ERROR(node_->get_logger(), " The joint states topic cannot be an empty string");
+    }
     else
-      joint_state_subscriber_ = nh_.subscribe(joint_states_topic, 25, &CurrentStateMonitor::jointStateCallback, this);
+      joint_state_subscriber_ = node_->create_subscription<sensor_msgs::msg::JointState>(
+          joint_states_topic,
+          std::bind(&planning_scene_monitor::CurrentStateMonitor::jointStateCallback, this, std::placeholders::_1),
```

Comment:
Please change to lambda and add brackets

Reply:
Could you please elaborate more .? I didn't understand what you meant